pinot-core/src/main/java/org/apache/pinot/core/operator/filter/predicate/PredicateEvaluator.java

```diff
@@ -92,6 +93,65 @@ default int applySV(int limit, int[] docIds, int[] values) {
     return matches;
   }
 
+  /**
+   * Apply the predicate to a batch of single-value entries.
+   * Compact matching entries into the prefix of the docIds array.
+   *
+   * @param limit How much of the input to consume.
+   * @param docIds The docIds associated with the values - may be modified by invocation.
+   * @param values Batch of dictionary ids or raw values.
+   * @param nullBitmap The document ids null bitmap.
+   * @return the index of the first non-matching entry.
+   */
+  default int applySV(int limit, int[] docIds, int[] values, ImmutableRoaringBitmap nullBitmap) {
```

Comment:
These changes are not needed. We can filter the docs in the DocIdIterator instead